tests/Microsoft.Identity.Test.Unit/WebUITests/WebView2WebUiFactoryTests.cs

```diff
@@ -1,25 +1,19 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-#if NET5_WIN 
+﻿#if DESKTOP || NET_CORE || NET5_WIN
```

Comment:
Merged `NetCoreWebUIFactoryTests` and `MsalDesktopWebUiTests` `WebView2WebUiFactoryTests`.